File: Examples/VisualBasic/AsposePdfGenerator/General/HelloWorldUsingAPI.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports AP = Aspose.Pdf

Namespace AsposePdfGenerator.General
    Public Class HelloWorldUsingAPI
        Public Shared Sub Run()
            Try
                ' ExStart:HelloWorldUsingAPI
                ' The path to the documents directory.
                Dim dataDir As String = RunExamples.GetDataDir_AsposePdfGenerator_General()

                ' Create pdf document
                Dim pdf1 As New AP.Generator.Pdf()

                ' Instantiate License class and call its SetLicense method to use the license
                Dim license As New AP.License()
                license.SetLicense("Aspose.Pdf.lic")

                ' Add a section into the pdf document
                Dim sec1 As AP.Generator.Section = pdf1.Sections.Add()

                ' Add a text paragraph into the section
                sec1.Paragraphs.Add(New AP.Generator.Text("Hello World"))

                dataDir = dataDir & Convert.ToString("HelloWorldUsingAPI_out_.pdf")
                ' Save the document
                pdf1.Save(dataDir)
                ' ExEnd:HelloWorldUsingAPI
            Catch ex As Exception
                Console.WriteLine(ex.Message + vbLf & "This example will only work if you apply a valid Aspose License. You can purchase full license or get 30 day temporary license from http:// Www.aspose.com/purchase/default.aspx.")
            End Try

        End Sub
    End Class
End Namespace

File: Examples/CSharp/AsposePdfGenerator/General/HelloWorldUsingAPI.cs
using System;
using System.IO;
using Aspose.Pdf;

namespace Aspose.Pdf.Examples.CSharp.AsposePdfGenerator.General
{
    public class HelloWorldUsingAPI
    {
        public static void Run()
        {
            try
            {
                // ExStart:HelloWorldUsingAPI
                // The path to the documents directory.
                string dataDir = RunExamples.GetDataDir_AsposePdfGenerator_General();

                // Create pdf document
                Aspose.Pdf.Generator.Pdf pdf1 = new Aspose.Pdf.Generator.Pdf();

                // Instantiate License class and call its SetLicense method to use the license
                Aspose.Pdf.License license = new Aspose.Pdf.License();
                license.SetLicense("Aspose.Pdf.lic");

                // Add a section into the pdf document
                Aspose.Pdf.Generator.Section sec1 = pdf1.Sections.Add();

                // Add a text paragraph into the section
                sec1.Paragraphs.Add(new Aspose.Pdf.Generator.Text("Hello World"));           

                dataDir = dataDir + "HelloWorldUsingAPI_out.pdf";
                // Save the document
                pdf1.Save(dataDir);
                // ExEnd:HelloWorldUsingAPI
            }
            catch (Exception ex)
            {
                Console.WriteLine(ex.Message + "\nThis example will only work if you apply a valid Aspose License. You can purchase full license or get 30 day temporary license from http:// Www.aspose.com/purchase/default.aspx.");
            }
            
        }
    }
}